webapp/client/cypress/integration/merkzeichenPersonA.spec.js

```diff
@@ -0,0 +1,124 @@
+describe("merkzeichenPersonA", () => {
+  beforeEach(() => {
+    cy.login();
+  });
+
+  context("with no data", () => {
+    beforeEach(() => {
+      cy.visit("/lotse/step/merkzeichen_person_a");
+    });
+
+    it("Should redirect to person a has disability page", () => {
+      cy.location().should((loc) => {
+        expect(loc.pathname.toString()).to.contain(
+          "/lotse/step/has_disability_person_a"
+        );
+      });
+    });
+  });
+
+  context("with person_a_has_disability no", () => {
+    beforeEach(() => {
+      cy.request("POST", "/testing/set_data/form_data", {
+        person_a_has_disability: "no",
+      });
+      cy.visit("/lotse/step/merkzeichen_person_a");
+    });
+
+    it("Should redirect to person a has disability page", () => {
+      cy.location().should((loc) => {
+        expect(loc.pathname.toString()).to.contain(
+          "/lotse/step/has_disability_person_a"
+        );
+      });
+    });
+  });
+
+  context("with person_a_has_disability yes", () => {
+    beforeEach(() => {
+      cy.request("POST", "/testing/set_data/form_data", {
+        person_a_has_disability: "yes",
+      });
+      cy.visit("/lotse/step/merkzeichen_person_a");
+    });
+
+    it("Should not stay on page and show error when submit button clicked", () => {
+      cy.get("button[type=submit]").click();
+      cy.url().should("include", "/lotse/step/merkzeichen_person_a");
+      cy.get("[role=alert][for=person_a_has_pflegegrad]").contains(
+        "Diese Angabe wird benötigt, um fortfahren zu können"
+      );
```

Comment:
I think you mean "Should stay on page and show error when submit button clicked"

Reply:
🙈